commands/generate.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Alex Ellis 2017. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/openfaas/faas-cli/schema"
+	"github.com/openfaas/faas-cli/stack"
+	"github.com/spf13/cobra"
+	yaml "gopkg.in/yaml.v2"
+)
+
+const (
+	functionNamespace = "openfaas-fn"
```

Comment:
Let's add the namespace as a flag with the default value `openfaas-fn`. 
We now support multiple OpenFaaS instances on Kubernetes and hardcoding the namespace will break this feature. cc @alexellis

Reply:
While doing it, I was even thinking the same. I will add that.

Reply:
Thank you

Reply:
I've run into a use case when is better to not have the namespace in there so it can be set with `kubectl -n`. So I think is better to have no default value and be omitted from the yaml output if not supplied.

```
Namespace string `yaml:"namespace,omitempty"`
```